src/harness/fourslashInterfaceImpl.ts

```diff
@@ -352,12 +352,12 @@ namespace FourSlashInterface {
             this.state.verifyBaselineFindAllReferences(...markerNames);
         }
 
-        public baselineGetFileReferences(fileName: string) {
-            this.state.verifyBaselineGetFileReferences(fileName);
+        public baselineFindAllReferencesMulti(seq: number, ...markerNames: string[]) {

```

Comment:
Introducing this helper for producing multiple baselines from the same test is really the only interesting part of this change.